api/data_explorer/controllers/export_url_controller.py

```diff
@@ -9,14 +9,18 @@
 import time
 import urllib
 
+from collections import OrderedDict
```

Comment:
I simulated exporting no cohort by checking all checkboxes in one facet.

Bad Gateway
The proxy server received an invalid response from an upstream server.
Apache Server at rawls.dsde-prod.broadinstitute.org Port 443

Shouldn't block this PR, but ideally we should fix this at some point

Reply:
Selecting all checkboxes works and correctly exports sample set containing all samples.